docs/features/highlights.md

```diff
@@ -20,13 +20,30 @@ There are currently three types of highlights you can add to your profile:
 
 ## How to Add a Highlight
 
-- Go to [OpenSauced's Highlights feed](https://app.opensauced.pizza/feed).
-- Click the "Post a highlight to show your work!" input.
+First, go to [OpenSauced's Highlights feed](https://app.opensauced.pizza/feed), then click the "Post a highlight to show your work!" input.
+
+There are two ways to adding highlights:
+
+### 1. Using Highlight Suggestions
+
+- Search the pull request or issue that you want to add in the "Highlight suggestions" section.
+- Click the "Fill content" button to paste the suggested link or the "Add and Summarize" button to paste the link and auto-summarize the content.
+
+![highlights demo](../../static/gif/highlight.gif)
+
+:::note
+
+The "Highlight suggestions" are only suggesting issues and pull requests that you created. For blog posts, you need to add them manually.
+
+:::
+
+### 2. Adding a Link Manually
+
 - Paste the URL to your blog post, pull request, or issue in the bottom input.
 - Either use our Auto-Summarize feature or write your own summary.
 
-![highlights demo](../../static/gif/highlight.gif)
+---
 
 For more examples of highlights, go to the [OpenSauced Highlights feed](https://app.opensauced.pizza/feed).
 
-Eager to get started? Check out the [Effectively Highlight Your Contribution](https://opensauced.pizza/learn/#/06-the-secret-sauce?id=effectively-highlight-your-contributions) section in our free "Intro to Open Source" course to learn more.
+Eager to get started? Check out the [Effectively Highlight Your Contribution](https://intro.opensauced.pizza/#/06-the-secret-sauce?id=effectively-highlight-your-contributions) section in our free "Intro to Open Source" course to learn more.
```

Comment:
Let's replace this link with `https://opensauced.pizza/learn/intro-to-oss/the-secret-sauce#effectively-highlight-your-contributions`. That's the most updated structure for our courses.

This actually catches a bug. We update the paths everywhere, but they're likely broken because the links should specify which course after /learn.

Reply:
Replaced. Thanks for reviewing :)